content/news/XXXX-XX-XX-jack-zero-latency.md

```diff
@@ -0,0 +1,44 @@
+title: "Has JACK zero latency?"
+authors: Daniel Schürmann
```

Comment:
Please add yourself to the `AUTHORS_METADATA` dict in the `pelicanconf.py`: https://github.com/mixxxdj/website/blob/website/pelicanconf.py#L158

Then run `tools/update_avatars.py`. It will automatically fetch your profile picture from github and add it to the source tree (don't forget to `git add` it).

Something like:

```python
AUTHOR_METADATA = {
    # ...
    "daschuer": {
        "name": "Daniel Schürmann", 
        "github": "daschuer",
        "discourse": "daschuer", 
        "tagline": "Mixxx Core Developer", 
    },
}
```

The author information including the avatar will be displayed below the post.